### PR TITLE
[v2] libcheck: Add check package

### PR DESCRIPTION
**Pull request by mlhs19**
Add check package.
Debian package (usr/bin/checkmk) depends on mawk.
but gawk is unsupported by meta-debian(-extended)?
So replaced mawk to gawk.

In Debian packages it is "check" but in Yocto's recipe it is "libcheck".

### CODE REVIEW

**Comment by masami256** [comment]
@mlhs19 Is it okay to close #52 ?

**Comment by mlhs19** [comment]
@masami256 
OK!

File: recipes-debian/libcheck/libcheck_debian.bb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# base commit: 21360dd971b39306b0332212a36ba9211f66e3a1
+
+SUMMARY  = "Check - unit testing framework for C code"
+HOMEPAGE = "https://libcheck.github.io/check/"
+SECTION = "devel"
+
+LICENSE  = "LGPLv2.1+"
+LIC_FILES_CHKSUM = "file://COPYING.LESSER;md5=2d5025d4aa3495befef8f17206a5b0a1"
+
+inherit debian-package
```
Comment (masami256) [inline recipes-debian/libcheck/libcheck_debian.bb:12]: @mlhs19 line 12,13 and line 25,26 are duplicated. Please delete ether.

**Comment by masami256** [comment]
> @masami256
> OK!

thank you! I closed it.

File: recipes-debian/libcheck/libcheck_debian.bb (hunk 1 above)
Comment (mlhs19) [reply recipes-debian/libcheck/libcheck_debian.bb:12]: I removed 25,26. force pushed.

Comment (masami256) [reply recipes-debian/libcheck/libcheck_debian.bb:12]: > I removed 25,26. force pushed.

@mlhs19 It seems your push https://github.com/miraclelinux/meta-debian-extended/commit/392a7ee46c7297145fe0c12429ca9747b8b4badc didn't remove line 25,26. Would you check it please?

Comment (mlhs19) [reply recipes-debian/libcheck/libcheck_debian.bb:12]: Sorry. I forgot to add. force pushed as 29bcf16

Comment (masami256) [reply recipes-debian/libcheck/libcheck_debian.bb:12]: thanks

**Pull request merged**